integrations/tensorflow/bindings/python/iree/tf/support/module_utils.py

```diff
@@ -893,23 +900,35 @@ class BackendInfo:
       "iree_vulkan": {
           "compiled_module_class": IreeCompiledModule,
           "driver": "vulkan",
-          "compiler_targets": ["vulkan-*"]
+          "compiler_targets": ["vulkan-spirv"]
       },
       "iree_llvmaot": {
           "compiled_module_class": IreeCompiledModule,
           "driver": "dylib",
           "compiler_targets": ["dylib-llvm-aot"]
       },
+      "iree_dylib_sync": {
+          "compiled_module_class": IreeCompiledModule,
+          "driver": "dylib-sync",
+          "compiler_targets": ["dylib-llvm-aot"]
+      },
   }
 
-  def __init__(self, backend_name: str, backend_id: str = None):
+  def __init__(self, backend_name: str,
+               backend_id: str = None,
+               compilation_flags: Sequence[str] = (),
+               runtime_flags: Sequence[str] = ()):
     """Creates a BackendInfo with the compilation details for backend_name.
 
     Args:
       backend_name: a str specifying which backend to use. Should be one of
         'tf', 'tflite', 'iree_vmla', 'iree_vulkan', 'iree_llvmaot'.
       backend_id: an optional str specifying what name to use when saving
         compiled artifacts. Must satisfy `backend_id.startswith(backend_name)`.
+      compilation_flags: an optional list of flags to pass to the compiler
+        when compiling artifacts for the backend.
+      runtime_flags: an optional list of flags to pas to the runtime when
+        running the generated artifacts for the backend.
```

Comment:
They are tuples, not lists.

note: empty tuple is OK since tuples are immutable. list is bad because If the function modifies the object (e.g., by appending an item to a list), the default value is modified.

Reply:
Good point. I recall you were bitten by this previously?